cmd/init.go

```diff
@@ -385,6 +385,7 @@ func init() {
 
 	// Use these flags to prepare the basecamp metadata json
 	initCmd.Flags().String("bgp-asn", "65533", "The autonomous system number for BGP conversations")
+	initCmd.Flags().String("bgp-cmn-asn", "65536", "The autonomous system number for CMN BGP clients")
```

Comment:
nit: does 65536 print off in the help or should we add a parenthetical stating the default?

Reply:
yes, it shows in the help
```
redbull-ncn-m001-pit:~/johren # csi config init --help | grep asn
      --bgp-asn string                        The autonomous system number for BGP conversations (default "65533")
      --bgp-cmn-asn string                    The autonomous system number for CMN BGP clients (default "65536")
      ```